### PR TITLE
Clean up the code by removing SQL

### PR DESCRIPTION
**Pull request by pkhalaj**
This is the preparation step for adding the comprehensive API, including tests and documentation.
SQL related files and references have been removed as they are no longer required at this stage.

### CODE REVIEW

File: changelog.rst [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+v?.?.? (2024-04-04)
```
Comment (pkhalaj) [inline changelog.rst:4]: We need to decide on the versioning as the tagged version on master is `0.1`.

Comment (mraspaud) [reply changelog.rst:4]: Good point, I should rénale the tag to 0.0.1.

File: changelog.rst [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+-------------------
+- Cleanup: remove all SQL related files and references
```
[the remark this answers is not in the document]
Comment (pkhalaj) [reply changelog.rst:6]: That's great. I agree to follow the same thing! 👍

**Comment by pkhalaj** [comment]
BTW, since I do not have write access to merge the PR, please fill free to merge it whenever you deem appropriate. 👍

**Pull request merged**